docs/service-brokers/docs/020-architecture-reb.md

```diff
@@ -3,25 +3,17 @@ title: The Remote Environment Broker architecture
 type: Architecture
 ---
 
-The Remote Environment Broker (REB) workflow starts with the registration process, during which a remote environment (RE) is registered on the Kyma cluster.
+The Remote Environment Broker (REB) workflow consists of the following steps:
 
-### Remote environment registration process
+1. The Remote Environment Broker watches for RemoteEnvironments (REs) in the cluster and EnvironmentMappings (EMs) in all Namespaces.
+2. The user creates an EnvironmentMapping custom resource in a given Environment. The EnvironmentMapping activates the services offered by a given RemoteEnvironment and has the same name as the RemoteEnvironment it corresponds to.
+3. The Remote Environment Broker creates a `remote-env-broker` Service Broker (SB) inside an Environment in which the EnvironmentMapping is created. This Service Broker contains data of all services provided by the RemoteEnvironments activated by EnvironmentMappings. There is always only one Service Broker per Environment, even if there are more EnvironmentMappings.
```

Comment:
`This Service Broker contains data of all services provided by the RemoteEnvironments activated by EnvironmentMappings.`

`This Service Broker contains data of all services provided by the activated RemoteEnvironments.`